views/components/programmes.njk

```diff
@@ -29,7 +29,7 @@
         {% endif %}
         {% if fields.applicationDeadline %}
             <dt>{{ __('global.programmes.programmeLabels.applicationDeadline') }}</dt>
-            <dd>{{ fields.applicationDeadline }}</dd>
+            <dd>{{ fields.applicationDeadline | safe }}</dd>
```

Comment:
This is because they want a linebreak in that field but I can't change this in Craft until this fix goes out.